task1/src/main/java/com/fundacionjala/movies/NewReleasePrice.java

```diff
@@ -0,0 +1,27 @@
+package com.fundacionjala.movies;
+
+/**
+ * Created by Mijhail on 11/06/2016.
+ */
+public class NewReleasePrice extends Rental {
```

Comment:
I think that it is not a good option to extend from Rental class.
Probably you should change the idea and have an Abstract Movie class to do tha